ipfs-api-prelude/src/api.rs

```diff
@@ -103,6 +103,36 @@ pub trait IpfsApi: Backend {
         self.request(add, Some(form)).await
     }
 
+    /// Add files using multipart::Form
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use ipfs_api;
+    /// use common_multipart_rfc7578::client::multipart;
+    ///
+    /// let add = ipfs_api::request::Add::builder()
+    /// .wrap_with_directory(true)
```

Comment:
Question: will this always need `wrap_with_directory` set to true when using `add_with_form`?

Reply:
I my use case yes, but I hesitate to say definitively there's no use case that doesn't use wrap_with_directory.